src/colmap/scene/synthetic.cc

```diff
@@ -73,13 +93,19 @@ void SynthesizeExhaustiveMatches(Reconstruction* reconstruction,
         }
       }
 
+      FeatureMatches matches = two_view_geometry.inlier_matches;
+      AddOutlierMatches(
+          options.inlier_match_ratio, num_points2D1, num_points2D2, &matches);
+
+      database->WriteMatches(image1.ImageId(), image2.ImageId(), matches);
       database->WriteTwoViewGeometry(
           image1.ImageId(), image2.ImageId(), two_view_geometry);
     }
   }
 }
 
-void SynthesizeChainedMatches(Reconstruction* reconstruction,
+void SynthesizeChainedMatches(const SyntheticDatasetOptions& options,
```

Comment:
Only ``inlier_match_ratio`` is used in this function. Is it necessary to pass a whole ``SyntheticDatasetOptions`` here?

Reply:
OK, done.